tests/kubevirt_api_lifecycle_automation_test.go

```diff
@@ -0,0 +1,184 @@
+package tests
+
+import (
+	"context"
+	"encoding/xml"
+	"fmt"
+	"github.com/kubevirt/kubevirt-api-lifecycle-automation/pkg/kubevirt-api-lifecycle-automation/resources/operator"
+	"github.com/kubevirt/kubevirt-api-lifecycle-automation/pkg/libvmi"
+	"github.com/kubevirt/kubevirt-api-lifecycle-automation/pkg/util"
+	libvirtxml "github.com/kubevirt/kubevirt-api-lifecycle-automation/tests/capabilities"
+	"github.com/kubevirt/kubevirt-api-lifecycle-automation/tests/framework"
+	"github.com/kubevirt/kubevirt-api-lifecycle-automation/tests/libnode"
+	"github.com/kubevirt/kubevirt-api-lifecycle-automation/tests/libvmifact"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gstruct"
+	gomegatypes "github.com/onsi/gomega/types"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	virt1 "kubevirt.io/api/core/v1"
+	"sync"
+	"time"
+)
+
+var _ = Describe("Kubevirt Api Lifecycle Automation tests", func() {
+	f := framework.NewFramework("job-test")
+	var kubevirtNS string
+	var once sync.Once
+	Context("Kubevirt Api Lifecycle Automation Job", func() {
+		BeforeEach(func() {
+			once.Do(func() {
+				c, err := f.GetVirtClient()
+				Expect(err).ToNot(HaveOccurred())
+				kv, err := c.KubevirtV1().KubeVirts(v1.NamespaceAll).List(context.Background(), v1.ListOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(kv.Items).To(HaveLen(1))
+				kubevirtNS = kv.Items[0].Namespace
+			})
```

Comment:
@Barakmor1 I would suggest to do it before each test in order to make sure the cluster is healthy for running the e2e